Source/ChocolateyGui/Properties/Resources.sv-SE.resx

```diff
@@ -0,0 +1,357 @@
+    <value>Senast uppdaterad</value>
+  </data>
+  <data name="PackageView_License" xml:space="preserve">
+    <value>Licens</value>
+  </data>
+  <data name="PackageView_Maintainers" xml:space="preserve">
+    <value> | Ansvariga: </value>
+  </data>
+  <data name="PackageView_PackageID" xml:space="preserve">
+    <value>Paket-ID</value>
+  </data>
+  <data name="PackageView_PackageSize" xml:space="preserve">
+    <value>Paketstorlek</value>
+  </data>
+  <data name="PackageView_ProjectSite" xml:space="preserve">
+    <value>Projektsida</value>
+  </data>
+  <data name="PackageView_ReleaseNotes" xml:space="preserve">
+    <value>Ändringslogg</value>
+  </data>
+  <data name="PackageView_ReportAbuse" xml:space="preserve">
+    <value>Rapportera missbruk</value>
+  </data>
+  <data name="PackageView_Summary" xml:space="preserve">
+    <value>Sammanfattning</value>
+  </data>
+  <data name="PackageView_TotalDownloads" xml:space="preserve">
+    <value>Totalt antal nedladdningar</value>
+  </data>
+  <data name="RemoteSourceView_Authors" xml:space="preserve">
+    <value>Av:</value>
+  </data>
+  <data name="RemoteSourceView_ButtonGoBackAPage" xml:space="preserve">
+    <value>Gå tillbaka en sida</value>
+  </data>
+  <data name="RemoteSourceView_ButtonGoForwardAPage" xml:space="preserve">
+    <value>Nästa</value>
+  </data>
+  <data name="RemoteSourceView_ButtonGotoFirstPage" xml:space="preserve">
+    <value>Gå till första sidan</value>
+  </data>
+  <data name="RemoteSourceView_ButtonGotoLastPage" xml:space="preserve">
+    <value>Sista</value>
+  </data>
+  <data name="RemoteSourceView_ButtonRefreshPkgs" xml:space="preserve">
+    <value>Kontrollera paket</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxAllVersions" xml:space="preserve">
+    <value>Alla versioner</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxIncludePrerelease" xml:space="preserve">
+    <value>Inklusive förhandsversioner</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxMatchExactly" xml:space="preserve">
+    <value>Matcha ord exakt</value>
+  </data>
+  <data name="RemoteSourceView_Downloads" xml:space="preserve">
+    <value>nedladdningar</value>
+  </data>
+  <data name="RemoteSourceView_SearchBoxText" xml:space="preserve">
+    <value>Sök:</value>
+  </data>
+  <data name="Resources_ThisPC" xml:space="preserve">
+    <value>Den här datorn</value>
+  </data>
+  <data name="SettingsView_About" xml:space="preserve">
+    <value>Om</value>
+  </data>
+  <data name="SettingsView_ButtonCancel" xml:space="preserve">
+    <value>Avbryt</value>
+  </data>
+  <data name="SettingsView_ButtonNew" xml:space="preserve">
+    <value>Ny</value>
+  </data>
+  <data name="SettingsView_ButtonRemove" xml:space="preserve">
+    <value>Ta bort</value>
+  </data>
+  <data name="SettingsView_ButtonSave" xml:space="preserve">
+    <value>Spara</value>
+  </data>
+  <data name="SettingsView_ChocoGui" xml:space="preserve">
+    <value>Chocolatey Gui</value>

```

Comment:
I think we should keep the casing as in the English source text where applyable.